.github/workflows/ci.yaml

```
@@ -0,0 +1,23 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12' # for firebase functions
+      - uses: actions/cache@v2
+        with:
+          # see: https://github.com/actions/cache/blob/main/examples.md#macos-and-ubuntu
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: npm i
```

`npm ci` (多分 clean install の略) だと `package.lock.json` にあるバージョンをそのままインストールしてくれるのでビルド前は ci でやりたいです〜

```suggestion
      - run: npm ci
```

---

ありがとうございます！